src/services/CreateAnalysisService.tsx

```diff
@@ -20,23 +20,30 @@ export interface AnalyzedImageResult {
 class CreateAnalysisService {
 
   static formatDate(dateStr: string): string {
```

Comment:
We could also maybe use the built-in date functions for format this for us.

For example
```
new Date(Date.parse(dateStr)).toLocaleDateString()
```
Although, this will give us a date in the format `MM/DD/YYYY`

Reply:
@colinywu @janakitti Had a similar comment on lines 24-39, please make sure to take a look there as well.